api/v1beta1/aerospikecluster_types.go

```diff
@@ -136,6 +136,8 @@ func (c *AerospikeOperatorClientCertSpec) validate() error {
 
 // AerospikePodSpec contain configuration for created Aeropsike cluster pods.
 type AerospikePodSpec struct {
+	// MetaData to add to pods.
```

Comment:
Do not use this metadata object. Create a new custom metadata struct having only annotations and labels

type ObjectMeta struct {
	Annotations 
	Labels
}

Reply:
Created new MetaObject